pixi.toml

```
@@ -0,0 +1,18 @@
+[project]
+name = "rattler"
+version = "0.7.0"
+description = "Rust library to install conda environments"
+authors = ["Wolf Vollprecht <wolf@prefix.dev>", "Bas Zalmstra <bas@prefix.dev>", "Tim de Jager <tim@prefix.dev>", "Ruben Arts <ruben@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
+build = "cargo build"
+
+[dependencies]
+cxx-compiler = "1.6.0.*"
```

Instead of `.*` I prefer to use `~=` to make the version range a bit more "fluid". (`~=` is the same as `>=1.2.3, <1.3`)

```suggestion
cxx-compiler = "~=1.6.0"
```

---

I'll definitely accept the change, but just to verify for my understanding, is that `>=1.2.3,<1.3` equivalent to `~=1.2.3`?

---

yes! 👍